### ISSUE TITLE
Remove/Upgrade Translator Project

### ISSUE THREAD

**Issue by jpgough-ms** [finos/architecture-as-code#717]
### Steps to Reproduce:

Dependencies and builds are stale in the translator project.

### Expected Result:

Builds are failing due to out of date dependencies

### Actual Result:

Build should pass

@Budlee and @matthewgardner should we remove or upgrade this component?

**Comment by Budlee** [comment]
Happy to fix the upgrades, I can do that in the next few days.

Originally we had this project to have a way to translate CALM to C4 in order to have a visualisation. I see this functionality moving into other parts of the project, specifically around the vscode extension. I think that a shared TypeScript library where the functionality for visualisation can as a better option

@matthewgardner what do you want to do with the K8s resource generation. I do see it as useful functionality still.

**Comment by jpgough** [comment]
@Budlee we merged a PR yesterday #720 - there’s one more library with an issue, but we might need spring parent bom updates to apply. 

@willosborne had a prototype of k8s generation working for a demo last year. @rocketstack-matt perhaps we should demo/discuss this in the next office hours?

**Comment by rocketstack-matt** [comment]
Agreed . . . I have been in favour of moving into the CLI, which would necessitate a re-write, although I do think now we have CALM Hub there is a supportable path forward for the Java version.

**Comment by Budlee** [comment]
@jpgough , missed your reply. I'll take a look at the dependency.

@rocketstack-matt, shall we make an issue spike out moving the functionality to the CLI? I remember I had a look at Typescript integration with C4 Structurizer and it was not there when I first created it. Maybe this has changed. Else we offer the functionality as part of the CALMHub?

**Comment by Budlee** [comment]
@jpgough I see what you mean that there needs to be an upgrade from spring, the fix release date is set for the 25th.  (https://github.com/spring-projects/spring-boot/issues/43568)

We can wait for that or just manually override the dependency, any preferance

**Comment by rocketstack-matt** [comment]
Sorry, you're right - my bad!
